src/controllers/wishlistController.js

```diff
@@ -1,5 +1,31 @@
 const mongoose = require('mongoose');
 const Wishlist = require('../models/Wishlist');
+const { StatusCodes } = require('http-status-codes');
+
+//Create wishlist
+const createWishlist = async (req, res) => {
+  const { albumId } = req.body; //frontend sends the albumId in the request body
```

Comment:
For creating the wishlist I'm not sure if it makes sense for the frontend to be _required_ to send a single album when first creating the wishlist :thinking: . 

What we may want to do 

*   create a wishlist automatically when user first registers
*   create a wishlist automatically if one doesn't alreay exist when user logs in
*   have an endpoint that allows a user to fetch their own wishlist, and if one doesn't exist it will be created on-the-fly

I'll write tickets to flesh those out a bit more.

In the meanwhile for this ticket, let's just create an empty wishlist (if the user doesn't already have one). Then the frontend can make subsequent calls to the "add album to wishlist" to add items after the wishlist has been successfully created.

Reply:
Thank you Akos, I have updated the create wishlist function, could you please check ? commit 10962af5bdcb4d17c5e054b3207777865192fd48